test/index_tests.js

```diff
@@ -204,4 +206,61 @@ describe('index.js', () => {
     });
   });
 
+});
+
+it('WHEN package json includes baseUrlPath THEN spec description is updated', done => {
+  const path = '/hello';
+  const newTitle = 'New title';
+  const app = express();
+  const pkgJsonWithBaseUrlPath = 'test/specs/withBaseUrlPath';
+  generator.setPackageInfoPath(pkgJsonWithBaseUrlPath);
+  generator.init(app, function(spec) {
+    spec.info.title = newTitle;
+    return spec;
+  });
+  app.get(path, (req, res, next) => {
```

Comment:
I think you can call `generator.getSpec()` here and it will contain description with or without 'base url' part already.

 ```suggestion
const spec = generator.getSpec();
assert.equal(spec.info.description.indexOf(', base url :') > 0, true);
```

Same for the test below.

Reply:
this fix did not work for me as it did not trigger a load of the packageInfo

Reply:
ok will try it by myself and then will merge the PR

Reply:
actually load of the packageInfo happens in a second after initialisation https://github.com/mpashkovskiy/express-oas-generator/blob/8bd6d88d9fbd2b216f9d2e1ce6cb794735747dff/index.js#L190 so if you add setTimeout it should work:

```suggestion
setTimeout(() => {
  const spec = generator.getSpec();
  assert.equal(spec.info.description.indexOf(', base url :') > 0, true);
}, 1001);
```